editor/src/components/canvas/ui-jsx-canvas-renderer/jsx-element-child-to-text.ts

```diff
@@ -0,0 +1,133 @@
+import type { JSXElementChild } from '../../../core/shared/element-template'
+import { assertNever } from '../../../core/shared/utils'
+
+export function jsxElementChildToText(
```

Comment:
this is just `jsxElementChildToText` from`ui-jsx-canvas-element-renderer-utils.tsx`, moved verbatim to a dedicated file